### ISSUE TITLE
[BUG] Register-PnPAzureADApp - Request_BadRequest","message":"A null value was found with the expected type 'Edm.String[Nullable=False]'. The expected type 'Edm.String[Nullable=False]' does not allow null values.

### ISSUE THREAD

**Issue by khha** [pnp/powershell#3583]
### Reporting an Issue or Missing Feature
I try to create an new Azure appregistration with a certificate with command "Register-PnPAzureADApp". I remember this script was working before. Now it is not working anymore.

### Expected behavior 
I expect a new Azure appregistration with name "AzureFunctionAppSP3" with a certificate and api permission "Sites.Selected".

### Actual behavior 
This is the response in PowerShell:

```
Checking if application 'AzureFunctionAppSP3' does not exist yet...Success. Application 'AzureFunctionAppSP3' can be registered.
Register-PnPAzureADApp:
Line |
   2 |   Register-PnPAzureADApp -ApplicationName "AzureFunctionAppSP3" -Tenan …
     |   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | {"error":{"code":"Request_BadRequest","message":"A null value was found with the expected type 'Edm.String[Nullable=False]'. The expected type 'Edm.String[Nullable=False]' does not allow null values.","innerError":{"date":"2023-11-20T12:18:12","request-id":"53467660-7217-4a3e-8305-3c4e3ef86baf","client-request-id":"53467660-7217-4a3e-8305-3c4e3ef86baf"}}}

```

### Steps to reproduce behavior
See script below to reproduce:

```
 $password = Read-Host -Prompt "Enter certificate password" -AsSecureString
 Register-PnPAzureADApp -ApplicationName "AzureFunctionAppSP3" -Tenant myCompany.onmicrosoft.com -CertificatePassword $password -SharePointApplicationPermissions "Sites.Selected" -Interactive 
```

### What is the version of the Cmdlet module you are running?
2.2.0

### Which operating system/environment are you running PnP PowerShell on?
- [X ] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by gautamdsheth** [comment]
hello @khha , thanks for reporting this.

It has been already fixed in the latest nightly builds , please uninstall the 2.2.0 and install the nightly build. It will be available in 2.3.0 as well when we release it , currently no ETA

To download latest nightly build, you can check this link :

https://pnp.github.io/powershell/articles/installation.html#nightly-build

Closing it as duplicate of https://github.com/pnp/powershell/issues/3316

**Comment by khha** [comment]
Thanks @gautamdsheth , I just have another [issue when installing the nightly build](https://github.com/pnp/powershell/issues/3584).
